packages/apollo-federation/src/index.ts

```diff
@@ -1,2 +1,5 @@
+import "core-js/features/array/flat";
```

Comment:
@abernix is it sufficient to polyfill at the index and call it a day? I don't actually know the answer to this, but I'm skeptical. Previously, we were executing the `apollo-env` polyfill in multiple files.

Reply:
Yeah, that should be effectively the same as what was happening before since we re-export everything from `src/composition` and `src/service` anyway.

Is it ideal, though? No, far from it.  We'll fix this in AS3!